Tests/SwiftConcurrentSequenceTests/ConcurrentMapTests.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Created by Dan Federman on 4/30/22.
+//  Copyright © 2022 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS"BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftConcurrentSequence
+import XCTest
+
+final class ConcurrentMapTests: XCTestCase {
```

Comment:
my tests here are testing that our mapping is correct, not that we are in fact executing concurrently. I'm not really sure how to test the latter (nor am I convinced that writing a "is this method actually concurrent" test is high-value)